### PR TITLE
devnet: Enable Regolith from genesis when deploying the local devnet

### PR DESCRIPTION
**Pull request by ajsutton**
**Description**

op-e2e: Add test to reproduce eth_getLogs failure

**Tests**

Please describe any tests you've added. If you've added no tests, or left important behavior untested, please explain why not.

**Invariants**

For changes to critical code paths, please list and describe the invariants or key security properties of your new or changed code.

**Additional context**

Add any other context about the problem you're solving.

**Metadata**
- Fixes https://linear.app/optimism/issue/CLI-3555/error-from-eth-getlogs-when-regolith-enabled
- Fixes https://linear.app/optimism/issue/CLI-3561/eth-getlogs-fails-on-pre-bedrock-blocks

### CODE REVIEW

**Comment by s7v7nislands** [comment]
maybe also changed in getting-started.json?

**Comment by ajsutton** [comment]
> maybe also changed in getting-started.json?

Yeah I think that makes sense though probably worth doing in a separate PR just for better visibility.

**Review by mslipper — APPROVED** [review]

**Review by trianglesphere — APPROVED** [review]

**Review by roninjin10 — APPROVED** [review]

File: ops-bedrock/Dockerfile.l2 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM ethereumoptimism/op-geth:optimism
+FROM us-docker.pkg.dev/oplabs-tools-artifacts/images/op-geth:optimism
```
Comment (tynes) [inline ops-bedrock/Dockerfile.l2:1]: Is this change to the dockerfile something that we will be planning on keeping?

Comment (trianglesphere) [reply ops-bedrock/Dockerfile.l2:1]: yea, we're trying to migrate to `us-docker.pkg.dev` from dockerhub

**Pull request merged**